lib/classes/ConfigSchemaHandler/index.js

```diff
@@ -136,6 +136,15 @@ class ConfigSchemaHandler {
             this.serverless.cli.log(`  ${message}`, 'Serverless', { color: 'orange' });
           }
         }
+        if (this.serverless.configurationInput.configValidationMode === undefined) {
+          this.serverless.cli.log(' ');
```

Comment:
What's the reasoning behind these empty spaces logged before and after the deprecation? I believe we're not doing that in other situations

Reply:
I was actually just imitating the block below it:
```
this.serverless.cli.log(' ');
this.serverless.cli.log(
  'Learn more about configuration validation here: http://slss.io/configuration-validation',
  'Serverless',
  { color: 'orange' }
);
this.serverless.cli.log(' ');
```
Your right I shouldn't have put the one on the below it. Do you think we should keep the one on top?